src/components/editTranscript/EditTranscript.tsx

```diff
@@ -91,18 +94,33 @@ const EditTranscript = ({
         display="flex"
         flexDir="column"
       >
-        <Box my={2}>
-          <Button
-            colorScheme="gray"
-            onClick={() => setIsModalopen(true)}
-            size="xs"
-            ml="auto"
-            display="block"
-            variant="outline"
-          >
-            Restore Original
-          </Button>
-        </Box>
+        <Flex my={2} justifyContent={"space-between"}>
+          <Box>
+            <Button
+              colorScheme="orange"
+              onClick={openGuidelines}
+              size="xs"
+              ml="auto"
+              display="block"
+              variant="solid"
+            >
+              Review Guidelines
```

Comment:
I think that an icon like `FaBook` could be a good match here at the beginning of the text